src/v/cloud_topics/tests/end_to_end_test.cc

```diff
@@ -119,3 +122,89 @@ TEST_F(e2e_fixture, test_l0_path) {
         }
     }
 }
+
+TEST_F(e2e_fixture, timequery) {
```

Comment:
Just FYI that [kafka_delete_records_transport](https://github.com/redpanda-data/redpanda/blob/dev/src/v/kafka/server/tests/delete_records_utils.cc) exists if you want to use it for testing against DeleteRecords requests

Reply:
oh I do! I am a fixture test noob.